tests/Credentials/UserRefreshCredentialsTest.php

```diff
@@ -110,6 +110,15 @@ public function testInitalizeFromAFile()
             new UserRefreshCredentials('scope/1', $keyFile)
         );
     }
+
+    public function testGcloudWarning()
+    {
+        putenv('SUPPRESS_GCLOUD_CREDS_WARNING=false');
+        $keyFile = __DIR__ . '/../fixtures2' . '/gcloud.json';
+        $this->assertNotNull(
```

Comment:
I believe this test should expect `PHPUnit_Framework_Error_Warning` is thrown

Reply:
@dwsupplee 
Yeah I wanted to make sure the test fails with that revision, now it has correct tests. PTAL

Reply:
Gotcha :)